app/models/index.js

```diff
@@ -7,7 +7,16 @@ const dbConfig = require('../../config/db')[config.environment];
 const basename = path.basename(__filename);
 const db = {};
 
-const sequelize = new Sequelize(dbConfig.database, dbConfig.username, dbConfig.password, dbConfig);
+let sequelize = null;
+
+if (process.env.NODE_ENV === 'production') {
+  sequelize = new Sequelize(process.env.DATABASE_URL, {
+    dialect: 'postgres',
+    dialectOptions: {
+      ssl: { rejectUnauthorized: false }
+    }
+  });
+} else sequelize = new Sequelize(dbConfig.database, dbConfig.username, dbConfig.password, dbConfig);
```

Comment:
This can be done in config files, there is no need to make this explicit here

Reply:
Also, do not access an env var without the config file (it works just fine, but you may "lose" it in the code)

Reply:
I tried to create a user, password and put the heroku host but with all of that fails. I'll Keep trying!